buildenv/jenkins/omrbuild.groovy

```diff
@@ -11,13 +11,15 @@ def setBuildStatus(String message, String state, String sha) {
 }
 
 def defaultCompile = 'make -j4'
+def defaultReference = '/home/jenkins/omr'
```

Comment:
What about `def defaultReference = '${HOME}/gitcache/omr'` instead of hardcoding path? Or something like that, making it relative to wherever jenkins slave is located.

Reply:
That is a good idea. I think I skipped over it because I was in a rush to get the builds unblocked and because of the problems I had with the libpath on zos. I will give this a quick try.

Reply:
Ok that worked. Still doesn't work on zos. I checked my OpenJ9 script and on zos we use git in an `sh` step directly in order to use the reference repo. Since we don't have a problem with fetching on zos at the moment I think I prefer to just leave it blank on zos so the code is cleaner. Will cleanup and resubmit for review.

Reply:
My 1 objection would be that we don't need the extra dir (~/gitcache/omr). We can simply use 1 dir (~/gitcache). Since this is a bare repo, we can add several repos if we needed and they wouldn't collide.